airflow/config_templates/default_airflow.cfg

```diff
@@ -658,6 +658,18 @@ auto_refresh_interval = 3
 # Boolean for displaying warning for publicly viewable deployment
 warn_deployment_exposure = True
 
+# Number of dags to look back for dag audit log view
+audit_trail_query_days = 31
+
+# Types of events to ignore for dag audit log view
+# E.g. excluded_events = success,failed,skipped,landing_times
+excluded_events =
```

Comment:
We probably want to exclude all of the read-only events by default to cut down on noise.
<img width="139" alt="Screen Shot 2022-01-12 at 9 54 58 AM" src="https://user-images.githubusercontent.com/4600967/149164066-6ac3d6df-149a-48eb-b20f-4e6cca47b6fc.png">